_layouts/details.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html>
+<!-- head -->
+{% include head.html %}
+
+<body id="top">
+    <div class="title-cont">
+        <div class="inner">
+            {{ page.main_title }}
```

Comment:
this should go and be done in the markdown file

Reply:
Done.